src/gui/dialogs/unit_attack.cpp

```diff
@@ -182,7 +182,7 @@ void unit_attack::pre_show(window& window)
 		defender_tooltip << translation::dsgettext("wesnoth", "Weapon: ") << "<b>" << defw_name << "</b>" << "\n"
 			<< translation::dsgettext("wesnoth", "Damage: ") << defender.damage << "<i>" << defw_specials_dmg << "</i>" << "\n"
 			<< translation::dsgettext("wesnoth", "Attacks: ") << defender.num_blows <<  "<i>" << defw_specials_atk <<  "</i>" << "\n"
-			<< translation::dsgettext("wesnoth-lib", "Chance to hit") << ": " << font::span_color(d_cth_color) << defender.chance_to_hit << "%</span>"<<  "<i>" << defw_specials_cth << "</i>"
+			<< translation::dsgettext("wesnoth", "Chance to hit: ") << font::span_color(d_cth_color) << defender.chance_to_hit << "%</span>"<<  "<i>" << defw_specials_cth << "</i>"
```

Comment:
The problem is that `dsgettext` doesn't get parsed and added to the .pot file, and those files are completely overwritten so any hand-added things are removed.

Although the file declares `#define GETTEXT_DOMAIN "wesnoth-lib"`, it doesn't have any translatable strings at the moment. Just remove the `GETTEXT_DOMAIN` line (it will default to "wesnoth" without the line), and then change all of the `translation::dsgettext(string, string)` calls to use `_ (string)` instead.